packages/editor/src/lib/hooks/useCanvasEvents.ts

```diff
@@ -49,12 +51,20 @@ export function useCanvasEvents() {
 				lastX = e.clientX
 				lastY = e.clientY
 
-				editor.dispatch({
-					type: 'pointer',
-					target: 'canvas',
-					name: 'pointer_move',
-					...getPointerInfo(e),
-				})
+				// For tools that benefit from a higher fidelity of events,
+				// we dispatch the coalesced events.
+				let events: PointerEvent[] | React.PointerEvent<Element>[] = [e]
+				if (currentTool.useCoalescedEvents && e.nativeEvent.getCoalescedEvents) {
```

Comment:
Additional check to make sure the browser supports this api - it doesn't seem to support it when accessing a local server via an ip address (possibly in other cases as well).